ntbs-integration-tests/NtbsWebApplicationFactory.cs

```diff
@@ -27,6 +27,7 @@ protected override void ConfigureWebHost(IWebHostBuilder builder)
                 throw new InvalidOperationException("Cannot build the web host before the test class name has been set.");
             }
 
+            Audit.Core.Configuration.AuditDisabled = true;

```

Comment:
I think it's worth adding a comment here explaining that we need to switch off auditing for the database seeding, and briefly explain why